Automattic/jetpack
# Admin Page: fix image paths in "My Plan" screen

Follow-up from #30401

## Proposed changes:

Let's avoid double `/` in paths for icon images in the "My Plan" screen.

<img width="1118" alt="image" src="https://github.com/Automattic/jetpack/assets/426388/1a1b7396-2be1-4ede-85ff-9c86ca479601">

### Other information:


## Jetpack product discussion

* p1683903885678439-slack-C0299DMPG

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Jetpack > Dashboard > My Plan on a site connected to WordPress.com.
* Check that the product and plan icons are displayed properly.
* Check that the image paths do not include a double '/'.
* Try with different products and plans.
